File: app/src/main/java/com/estrrado/tradopay/ui/modules/auth/LoginActivity.java


package com.estrrado.tradopay.ui.modules.auth;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;

import com.estrrado.tradopay.ui.modules.MainActivity;
import com.example.tradopay.R;

public class LoginActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_login);

    }
    public void onSkipClick(View view)
    {
        startActivity(new Intent(LoginActivity.this, MainActivity.class));
    }
}

File: app/src/main/java/com/estrrado/tradopay/activity/LoginActivity.kt
package com.estrrado.tradopay.activity

import android.content.Intent
import android.os.Bundle
import android.view.View
import androidx.appcompat.app.AppCompatActivity

import com.example.tradopay.R

class LoginActivity : BaseActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
    }

    override fun initLayout() {
        setContentView(R.layout.activity_login)
    }

    override fun initListeners() {
        TODO("Not yet implemented")
    }

    override fun initObservers() {
        TODO("Not yet implemented")
    }

    fun onSkipClick(view: View?) {
        startActivity(Intent(this@LoginActivity, MainActivity::class.java))
    }
}